undo watermask revert (#257)
ASFHyP3/hyp3-gamma: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [4.7.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v4.6.0...v4.7.0)

### Added

> * Option to apply water masking to output geotiff files produced by gamma processing
>   * define the nodata value (minimum_value_datatype) to the dem.tif
>     and apply this dem.tif to gamma processing

Reviewer: ```suggestion
* Option to apply a water mask to DEM before processing and include the water mask GeoTIFF with InSAR products
```

Reviewer: okay, my mistake, the changlog as stated is more accurate.

### Changed
* `conda-env.yml` has been renamed to `environment.yml` in-line with community practice

> 

Author: ```suggestion
```
